base/sparse/sparsematrix.jl

```diff
@@ -795,7 +795,24 @@ end
 droptol!(A::SparseMatrixCSC, tol, trim::Bool = true) =
     fkeep!(A, (i, j, x) -> abs(x) > tol, trim)
 
+"""
+    dropzeros!(A::SparseMatrixCSC, trim::Bool = true)
```

Comment:
There is also `droptol!` for `SparseVector`, maybe change `SparseMatrixCSC` to `SparseArray` in the signatures`?